### PR TITLE
Test Branch for GitBranch placeholder

### PR DESCRIPTION
**Pull request by ajit-jain**

### CODE REVIEW

File: templates/shippable/gitBranchPlaceholderTemplate/templateDefinition.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      - name: PIPE_6522_p0001_step_002
+        type: Bash
+        configuration:
+          inputSteps:
+            - name: PIPE_6522_p0001_step_001
+        execution:
+          onExecute:
+            - printenv
+
+      {{ if eq "{{gitBranch}}" "master" }}
```
Comment (vishnurao12) [inline templates/shippable/gitBranchPlaceholderTemplate/templateDefinition.yml:32]: wont the pipeline sync fail?

File: values.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+GitRepo: 
+  gitProvider: my_github
+  path: ajit-jain/Org_test_Automation
```
Comment (vishnurao12) [inline values.yml]: should this be "jfbetapipeorg'

File: templates/shippable/gitBranchPlaceholderTemplate/templateDefinition.yml (hunk 1 above)
Comment (ajit-jain) [reply templates/shippable/gitBranchPlaceholderTemplate/templateDefinition.yml:32]: This is intended to pass in this issue.

File: values.yml (hunk 2 above)
Comment (ajit-jain) [reply values.yml]: Sorry, I am not using it . I need to remove this.

File: templates/shippable/gitBranchPlaceholderTemplate/values.yml.example [hunk 3]
```diff
@@ -0,0 +1,3 @@
+myRepo:
+  gitProvider: my_github
```
Comment (vishnurao12) [inline templates/shippable/gitBranchPlaceholderTemplate/values.yml.example]: s_gitHub

**Pull request merged**